### DISCUSSION TITLE
How can I open devtools in the middle of the test?

### DISCUSSION THREAD

**Discussion by Uno-count** [seleniumbase/SeleniumBase#2981] [Q&A] [answered]
I want to open devtools in the middle of the test and navigate to the console, is there a way to do that?

**Comment by mdmintz** [comment, marked as answer]
You can try using the `F12` Key: https://stackoverflow.com/a/13065150/7058266
Or just https://stackoverflow.com/a/68745841/7058266

Otherwise, use ``--devtools`` / ``devtools=True`` to start SeleniumBase with Devtools open.

**Reply by Uno-count** [reply]
I have tried this line of code but did not open 

self.action_chains.key_down(Keys.CONTROL).key_down(Keys.SHIFT).send_keys('j').key_up(Keys.CONTROL).key_up(Keys.SHIFT).perform()

**Reply by Uno-count** [reply]
I have solved it using this following code.

`import pyautogui`

`pyautogui.hotkey('command', 'option', 'j') ` used in Mac
`pyautogui.hotkey('ctrl', 'shift', 'j')` used in windows 

Thank you for your hard work @mdmintz 🚀
